.github/workflows/test-gnofract4d.yml

```diff
@@ -18,9 +18,6 @@ jobs:
           - os: ubuntu-18.04
             python: 3.6
             toxenv: py
-          - os: ubuntu-16.04
```

Comment:
Yes, 3.5 is the standard Python 3 for Ubuntu 16.04 so this does mean dropping it.